modules/misc/news.nix

```
@@ -1229,6 +1229,19 @@ in
           it anymore.
         '';
       }
+
+      {
+        time = "2019-11-11T10:27:48+00:00";
+        message = ''
+          A new module is available: 'xdg.mime'.
+
+          If enabled, this module will create the XDG mime database
```

Should indicate that the module is enabled by default. Perhaps something like
```suggestion
          If enabled, which it is by default, this module will create the XDG mime database
```
suffices?

---

Ah, good catch. I think I wasn't sure whether it should even be enabled by default, but it seems you don't mind. The commands' execution time does seem to be negligible.

---

My spontaneous reaction was that it should be disabled by default but I think it should be fine since the NixOS module seems to be enabled by default and I haven't heard any complaints about that.